lib/datasets/cache-path.rb

```diff
@@ -0,0 +1,32 @@
+module Datasets
+  class CachePath
+    def initialize(id)
+      @id = id
+    end
+
+    def base_dir
+      Pathname(system_cache_dir).expand_path + 'red-datasets' + @id
+    end
+
+    def remove
+      FileUtils.rmtree(base_dir.to_s, secure: true) if base_dir.exist?
+    end
+
+    def remove_file(file)
+      FileUtils.rm(base_dir + file) if (base_dir + file).exist?
+    end
```

Comment:
It is another option to implement this method in the Record class.
I wondered it, but decided on this.
How  about it?

Reply:
I think that we don't need to provide the API.
See also my comments in `Record`.